### ISSUE TITLE
Terms "disjoint/distinct variables"

### ISSUE THREAD

**Issue by avekens** [metamath/set.mm#2905]
Currently, two terms are used for the same concept: "disjoint/distinct variables"

On the Metamath Proof Explorer Home Page (see https://us.metamath.org/mpeuni/mmset.html#distinct), "distinct" is used ("distinct variables", "distinct variable condition"), but we have (at least) 11 "disjoint variable condition" in set.mm (and 106 "distinct variable condition"). There is, however, also the following hint: 

> ... This is why a distinct variable pair is officially called a "disjoint variable pair" in the Metamath specification.

@wlammen Proposed to use "disjoint", because of `$d x ph $.`. The error message of the metamath program also says "disjoint variable requirement".... 

I think there was already a discussion about these terms, but I do not know where and if there was an agreement which term to be used.

Finally, the term(s) "disjoint/distinct variables" are abbreviated as "DV" and "dv" in set.mm:
* "dv condition" 174 times,
* "DV condition" 30 times.
I would propose to unify it to "DV condition". This is also according to our conventions: "No two variables occurring in the LHS may share a disjoint variable (DV) condition."

(See also discussion in #2904).

**Comment by benjub** [comment]
I agree with @avekens and @wlammen to capitalize "DV condition" since it's an acronym. The "D" is for "disjoint"; I think "distinct" can be misleading (even if @digama0 uses it in https://arxiv.org/abs/1601.07699 and https://us.metamath.org/downloads/metamath.pdf uses both rather interchangeably). We can also add a paragraph in ~conventions, and link to it from ~ ax-5, which is the first statement with a DV condition.
Of course I'm a bit partial since I wrote that latest example that @avekens mentioned above, and I used "DV condition", meaning "disjoint", systematically in https://arxiv.org/abs/2202.10383

**Comment by benjub** [comment]
Just as a heads up to @avekens and @wlammen to avoid duplicate work: I will do that.

**Comment by benjub** [comment]
I did this in PR #2923, as promised. @jkingdon proposes there to spell out "DV".  I can do that, through a simple search and replace. I thought "DV" was ubiquitous enough to deserve an abbreviation, but indeed it might be a bit obscure to someone landing on a given page.  So no strong opinion.  What do you think, @avekens @wlammen ?

**Comment by avekens** [comment]
I think "DV" can and should be used as abbreviation, but, as I said before, should be unified with the more often used abbreviation "dv" ( I prefer the uppercase version). The abbreviation "dv" is introduced in the comment of ~stdpc5v - is this the right place?.

**Comment by benjub** [comment]
@avekens : so what you propose is the current state of #2923, correct ?
Then, as I wrote there, a future PR (which I will do) needs to document it better, with a paragraph in ~conventions and links to ~conventions and the home page from where it is first introduced, which is ax-5 for set.mm and ax-17 for iset.mm.

Note: one can search-and-replace easily between "DV condition" and "disjoint variable condition" quite easily, so that can also be done/undone/redone separately.

**Comment by wlammen** [comment]
I am more on the side of spelling it out.  The dv shorthand is used far too seldom to save any space of significant amount.  Do we use shorthands elsewhere, apart from the ubiquitous names?  On the first 5000 lines or so I just found the examples 'ASCII', 'monothetic BCI logic' and 'system DS of Vladimir Lifschitz'.  These are at least easy to look up in the internet because sufficient context is given.  But DV is our own creation, at least I cannot find anything pointing to 'disjoint variables' by Google search.  So what is the point in presenting a casual visitor of the Metamath site with some unexplained and in-group abbreviation?

**Comment by jkingdon** [comment]
> So what is the point in presenting a casual visitor of the Metamath site with some unexplained and in-group abbreviation?

Nicely said. And yes this is my argument for spelling it out. It is far more obscure than, say ZFC or even AC (although for the latter I would use "axiom of choice" or "choice" rather than the acronym). And yes, it is metamath specific because most works deal with free and bound variables rather than disjoint variable constraints as such.

As for "distinct" versus "disjoint" the former somehow makes more sense to me (I guess I'm more focused on the pairwise constraints even if a $d statement can have more than two variables). I agree that existing usage is to use both interchangeably so I suppose I'd keep that practice.

**Comment by avekens** [comment]
> As for "distinct" versus "disjoint" the former somehow makes more sense to me (I guess I'm more focused on the pairwise constraints even if a $d statement can have more than two variables). I agree that existing usage is to use both interchangeably so I suppose I'd keep that practice.

Yes, I think there is still no final decision/agreement if only "distinct" or only "disjoint" shall be used, or both interchangeably. If we take the Metamath Proof Explorer Home Page (see https://us.metamath.org/mpeuni/mmset.html#distinct) as reference, "distinct" should be used ("disjoint" is only used in the context of a pair of variables: "disjoint variable pair"), but according to the Metamath book, "disjoint" is used more often (see subsection "Verifying Disjoint Variable Restrictions" of section "4.1.4 Proof Verification") - here, "distinct" seems to be used only for setvar variables: 

> `$d x y $.` means "assume x and y are distinct variables."

According to the abbreviation, this is also introduced in the Metamath book, section C.2.4 Statements:

> We denote by DV the set of all unordered pairs { a , b } C_ VR such that a =/= b. DV stands for "distinct variables."

If we decide to use "distinct" and "disjoint" interchangeably, I think this would be more confusing for a reader not familiar with this concept than to use DV consistently and explain it's meaning in detail at a central place.

**Comment by benjub** [comment]
End result:
1. I will spell it out, i.e., consistently write "disjoint variable condition".
2. I will use "disjoint", as I wrote above, and I think eventually "disjoint" should replace "distinct" almost everywhere, since "distinct" can be misleading (see my argument above, or the arXiv paper I linked to; and the Metamath book uses "disjoint" when it wants to be more formal).

**Comment by benjub** [comment]
First phase done with the merging of #2923. Next phase (that I will do in the coming days), will be to add some documentation (mainly by linking to existing documentation from strategic places), taking into account useful remarks by multiple contributors here and in the comments of #2923.
